src/uu/cp/src/copydir.rs

```diff
@@ -93,7 +93,7 @@ impl<'a> Context<'a> {
     fn new(root: &'a Path, target: &'a Path) -> std::io::Result<Self> {
         let current_dir = env::current_dir()?;
         let root_path = current_dir.join(root);
-        let root_parent = if target.exists() {
+        let root_parent = if target.exists() && !root.to_str().unwrap().ends_with("/.") {
```

Comment:
Is this the only edge case in which the files are copied?

Reply:
I thought so from the original issue, but was so long ago I dont remember if there was something else present at the time :stuck_out_tongue:

Reply:
Oh yeah it is super long ago 😄

Reply:
Well, looks good. Sorry it took so long!